service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CoreCaseDataStoreLoader.java

```diff
@@ -25,6 +25,11 @@ public static CaseDetails populatedCaseDetails() throws IOException {
         return mapper.readValue(response, CaseDetails.class);
     }
 
+    public static CaseDetails reformRespondentCaseDetails() throws IOException {
```

Comment:
I can imagine when migration is done and we clean up old definitions we remove that method and add new structure to `core-case-data-store-api/populated-case-details.json`

Reply:
Yes - I did not want to break existing tests with this change.